Tools/CISupport/ews-build/config.json

```diff
@@ -433,7 +433,7 @@
       "builderNames": ["Unsafe-Merge-Queue"]
     },
     {
-      "type": "Periodic", "name": "safe-merge-queue", "periodicBuildTimer": 7200,
+      "type": "Periodic", "name": "safe-merge-queue", "periodicBuildTimer": 1800,
```

Comment:
I recommended running this bot every 30 minutes, @aj062, any thoughts?

Reply:
30 or even less, maybe 10 or 15 minutes. Considering that in future this could be the recommended approach for engineers to use safe-merge-queue by default, If someone adds this label on a PR with everything green, it would be wasteful to wait 30 minutes (in worst case) for almost no reason.

Reply:
@briannafan, if @aj062 want's to go more frequent, 15 minutes sounds reasonable to me.